includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -205,6 +205,7 @@ public function enqueue_scripts() {
 		wp_enqueue_style( 'wc-setup', WC()->plugin_url() . '/assets/css/wc-setup.css', array( 'dashicons', 'install' ), WC_VERSION );
 
 		wp_register_script( 'wc-setup', WC()->plugin_url() . '/assets/js/admin/wc-setup' . $suffix . '.js', array( 'jquery', 'wc-enhanced-select', 'jquery-blockui', 'wp-util', 'jquery-tiptip' ), WC_VERSION );
+		wp_style_add_data( 'wc-setup', 'rtl', 'replace' );
```

Comment:
Could you please move this line up and place it right after `wc-setup` style (and not the script) is enqueued?